### PR TITLE
Trash

### PR DESCRIPTION
**Pull request by tanishk15sharma**
### TRASH MANAGEMENT

> Style

- [x] Trash Card

> Functionility

- [x] Trash `useContext`
- [x] Trash `useState`
- [x] Delete Note to Trash
- [x] Restore Note From Trash
- [x] permanent Delete Note
- [x] Clear Trash

### CODE REVIEW

**Review by Hardiegogo — APPROVED** [review]
Clean code,
You should probably add preview links or screenshots for better understanding.

File: src/components/notes-listing/note-card/NoteCard.jsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import React, { useState, useRef, useEffect } from "react";
 import { useNotes } from "../../../context/NotesContext";
```
Comment (VedanthB) [inline src/components/notes-listing/note-card/NoteCard.jsx:2]: you can do a barrel import / export , makes the code look cleaner

**Review by VedanthB — APPROVED** [review]
Good work

things i liked

- clean code and organized folder structure 
- good variable naming

you can merge the PR

File: src/components/notes-listing/note-card/NoteCard.jsx (hunk 1 above)
Comment (tanishk15sharma) [reply src/components/notes-listing/note-card/NoteCard.jsx:2]: yes.

**Pull request merged**